### ISSUE TITLE
android faker causing deliveroo to crash

### ISSUE THREAD

**Issue by SquadCraftFTW** [Android1500/AndroidFaker#60]
Modules:
Kernelsu/apatch
Zygisknext latest
Lsposed
Android faker

Device: Xiaomi 13 ultra, Android 14/Hyperos

When I try to open the deliveroo app it either crashes or sends me to deliveroo support website, when I unselect deliveroo from the android faker app list in lsposed it opens normally again. 

App: https://play.google.com/store/apps/details?id=com.deliveroo.orderapp&pli=1

**Comment by SquadCraftFTW** [comment]
https://github.com/user-attachments/assets/bee8c10d-21c4-49f4-9b22-cbab11f6e3c9

Video of the issue.

**Comment by firmantsani** [comment]
> Screenrecorder-2024-10-23-02-20-46-513.mp4 
> Video of the issue.

Maybe, the app detects the hook, not because of the faker android app that is problematic

**Comment by komzishim** [comment]
Did you ever find a fix for this I am having the same issue

**Comment by SquadCraftFTW** [comment]
Yeah, deliveroo was detecting zygisk. Magisk alpha denylist with built in zygisk fixed it for me, I believe KSU with zygisk next and the root hiding template also works.

**Comment by SquadCraftFTW** [comment]
It does work, you can test by downloading android id app from play store. 

Also you're using this for deliveroo, are you using card payments because i'm having trouble getting deliveroo to work nowadays.

**Comment by komzishim** [comment]
For deliveroo I pay through google pay.
